source/templates/workflows.slim

```diff
@@ -197,13 +197,14 @@
 				h2 == data[:strings][:workflows][:inputs][:insert_variable][:action]
 				input[type="text" ng-model="insertVariableCtrl.keyFilter" placeholder="#{ data[:strings][:workflows][:inputs][:insert_variable][:key_filter_placeholder] }"]
 			article.vertical-scroll
-				p.progress-indicator[progress-model="insertVariableCtrl.loadInsertableVariablesProgress"]
-				div[ng-if="insertVariableCtrl.loadInsertableVariablesProgress.isIdle"]
-					ul.variable-source[ng-repeat="variablesOfSource in insertVariableCtrl.insertableVariables"]
-						li[ng-repeat="variable in variablesOfSource.variables | variablesfilteredByKey: insertVariableCtrl.keyFilter" ng-if="insertVariableCtrl.shouldShowVariable(variable)"]
-							button[ng-click="insertVariableCtrl.variableSelected(variable)"]
-								strong[ng-bind="variable | prettifiedVariableKey"]
-								em[ng-bind="variablesOfSource.title"]
+				.list-of-available-secrets
```

Comment:
Not secrets, this is the simple env var insert popup. Otherwise, working perfectly and looks good.